src/ws-inline-edit/ws-inline-edit.js

```diff
@@ -0,0 +1,95 @@
+import {React, Component} from '../imports';
+require('./ws-inline-edit.scss');
```

Comment:
Actually I didn't find where can I inject my styles xD
I think webpack should automay=te this bundling, but until I did it styles don't work